src/utils/parsers.ts

```diff
@@ -86,6 +86,9 @@ const parseNodeId: (data: string) => ids.NodeId = validateParserToArgParser(
 );
 const parseGestaltId: (data: string) => ids.GestaltId =
   validateParserToArgParser(ids.parseGestaltId);
+
```

Comment:
No whitespace, but you can have another commit that creates whitespace between all the definitions.

Reply:
Resolved